inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api_impl.cpp

```diff
@@ -523,23 +518,22 @@ void InferenceEnginePython::IEExecNetwork::createInferRequests(int num_requests)
         infer_request.request_queue_ptr = request_queue_ptr;
         infer_request.request_ptr = actual.CreateInferRequest();
 
-        infer_request.request_ptr.SetCompletionCallback<std::function<void(InferenceEngine::InferRequest r,
-                                                                            InferenceEngine::StatusCode)>>(
-                [&](InferenceEngine::InferRequest request, InferenceEngine::StatusCode code) {
-                    if (code != InferenceEngine::StatusCode::OK) {
-                        IE_EXCEPTION_SWITCH(code, ExceptionType,
-                                    InferenceEngine::details::ThrowNow<ExceptionType> {} <<=
-                                            std::stringstream {} << IE_LOCATION << InferenceEngine::details::ExceptionTraits<ExceptionType>::string());
-                    }
-
-                    auto end_time = Time::now();
-                    auto execTime = std::chrono::duration_cast<ns>(end_time - infer_request.start_time);
-                    infer_request.exec_time = static_cast<double>(execTime.count()) * 0.000001;
-                    infer_request.request_queue_ptr->setRequestIdle(infer_request.index);
-                    if (infer_request.user_callback) {
-                        infer_request.user_callback(infer_request.user_data, code);
-                    }
-                });
+        infer_request.request_ptr.SetCompletionCallback<std::function<void(InferenceEngine::InferRequest r, InferenceEngine::StatusCode)>>(
+            [&](InferenceEngine::InferRequest request, InferenceEngine::StatusCode code) {
+                if (code != InferenceEngine::StatusCode::OK) {
+                    IE_EXCEPTION_SWITCH(code, ExceptionType,
+                                        InferenceEngine::details::ThrowNow<ExceptionType> {} <<=
+                                        std::stringstream {} << IE_LOCATION << InferenceEngine::details::ExceptionTraits<ExceptionType>::string());
+                }
+
+                auto end_time = Time::now();
+                auto execTime = std::chrono::duration_cast<ns>(end_time - infer_request.start_time);
+                infer_request.exec_time = static_cast<double>(execTime.count()) * 0.000001;
+                infer_request.request_queue_ptr->setRequestIdle(infer_request.index);
```

Comment:
Am I right that you change only code style in this file?

Reply:
You are right, changes by Clang_format